starter-code/clue.js

```diff
@@ -102,3 +121,16 @@ function reveal_mistery(mistery) {
 
 var mistery_envelope = pick_mistery();
 reveal_mistery(mistery_envelope);
+
+
+
+function chooseYourDestiny()  {
```

Comment:
As it is written, you have a function definition (chooseYourDestiny) inside of another function definition (random_selector)... Make to keep track of your open and curly braces.